test/PensionFundReleaseTest.js

```diff
@@ -7,175 +7,229 @@ let Token = artifacts.require("zeppelin-solidity/contracts/token/SimpleToken.sol
+        fund = await PensionFundRelease.new.apply(
+            this,
+            deployParams(
+                firstPaymentTime,
+                firstPaymentPercent,
+                token.address,
+                PAYOUT_PERCENT
+            )
+        )
+        // Transfer previous balance away
+        await token.transfer(UNAUTHORIZED, INITIAL_BALANCE, {from: WORKER})
+        await token.transfer(fund.address, balance)
+        await fund.vote(true, "justification", {from: VALIDATORS[0]})
+        await fund.vote(true, "justification", {from: VALIDATORS[1]})
+        await fund.releaseRoots({from: WORKER})
+        workerBalance = await token.balanceOf(WORKER)
+        workerBalance.toNumber().should.be.equal(INITIAL_BALANCE)
+        let release = await fund.releaseRoots({from: WORKER})
+        release.logs[0].args.amount.toNumber().should.be.equal(0)
+    })
+
+    it("#13 should refund all roots over the time", async() => {
+        let masterBalance
+        let balance = INITIAL_BALANCE
+        // Transfer previous balance away
+        await token.transfer(UNAUTHORIZED, FIRST_PAYMENT_PERCENT)
+        await token.transfer(fund.address, balance)
+        await fund.vote(false, "justification", {from: VALIDATORS[0]})
+        await fund.vote(false, "justification", {from: VALIDATORS[1]})
+        let refund = await fund.refundRoots({from: MASTER})
+        while (await fund.balance() > 0) {
+            web3.currentProvider.send({
+                jsonrpc: '2.0',
+                method: 'evm_increaseTime',
+                params: [TIME_INCREMENT],
+                id: new Date().getTime()
+            })
+            refund = await fund.refundRoots({from: MASTER})
+            refund.logs[0].args.amount.toNumber().should.be.equal(PAYOUT_PERCENT)
+        }
+        masterBalance = await token.balanceOf(MASTER)
+        masterBalance.toNumber().should.be.equal(INITIAL_BALANCE)
+
+        // recover previous MASTER balance
+        let balanceForBurn = await token.balanceOf(MASTER).then((res) => {
+            return res.c[0]
         })
-        release = await fund.releaseRoots({from: WORKER})
-        release.logs[0].args.amount.toNumber().should.be.equal(PAYOUT_PERCENT)             
-      }
-      workerBalance = await token.balanceOf(WORKER)
-      workerBalance.toNumber().should.be.equal(INITIAL_BALANCE)   
+        await token.transfer(0x00, balanceForBurn , {from: MASTER})
     })
 
-    it("#12 should release all roots and then release balance of 0", async () => {
-      let workerBalance, balanceNumber
-      let firstPaymentPercent = PERCENT_DENOMINATOR
-      let error;
-      let balance = INITIAL_BALANCE 
-      fund = await PensionFundRelease.new.apply(
-        this,
-        deployParams(
-          firstPaymentTime,
-          firstPaymentPercent,
-          token.address,
-          PAYOUT_PERCENT
+    it("#14 should refund all roots and then refund balance of 0", async() => {
```

Comment:
Please, don't forget to change tests after your changes in PR.
You need just to check that all ether amount are refunded, if burn was approved.
And also a negative scenario (burn was not approved -> failure)